service/controller/v17/key/key.go

```diff
@@ -223,6 +227,21 @@ func IsPodDrained(pod *corev1.Pod) (bool, error) {
 	return b, nil
 }
 
+func KubeletVolumeSizeFromNode(node v1alpha1.KVMConfigSpecKVMNode) string {
+	// TODO: calvix
+	// TODO: for now we use same value as for DockerVolumeSizeFromNode, when we have kubelet size in spec we should use that.
```

Comment:
Can we make sure this is tracked and cross referenced?

Reply:
i put it here https://github.com/giantswarm/giantswarm/issues/4105#issuecomment-421772917

Reply:
We try to cross reference the issue tracking the TODO in the code so people finding TODOs in the code can check the issues. Otherwise you are just lost when you see TODOs of other people.